### PR TITLE
[8.0] Fix pep8

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: __unported__/base_delivery_carrier_label/stock.py [hunk 1]
```diff
@@ -310,8 +310,7 @@ def create(self, cr, uid, values, context=None):
         picking_obj = self.pool['stock.picking']
         values = picking_obj._values_with_carrier_options(cr, uid, values,
                                                           context=context)
-        picking_id = super(stock_picking_in, self
-                    ).create(cr, uid, values, context=context)
+        picking_id = super(stock_picking_in, self).create(cr, uid, values, context=context)
```
Comment (eLBati) [inline __unported__/base_delivery_carrier_label/stock.py:313]: Lines like this are longer than 80. But travis doesn't complain because within **unported**

Comment (bwrsandman) [inline __unported__/base_delivery_carrier_label/stock.py:313]: Max line length is set to 120

Comment (eLBati) [inline __unported__/base_delivery_carrier_label/stock.py:313]: Probably I missed the decision to increment the limit.
Has it been discussed somewhere?

Comment (bwrsandman) [inline __unported__/base_delivery_carrier_label/stock.py:313]: On the mailing list last year.
https://lists.launchpad.net/openerp-community/msg03986.html

I put 120 so we wouldn't ignore line length entirely, and now it is at least uniformly checked everywhere in OCA (no style splintering).

If we want to have the dicussion again and we come to a conclusion that the line length is, say 100 (suggested in [pep8](http://dinsdale.python.org/dev/peps/pep-0008/#maximum-line-length)) we have a constructive way of enforcing it.

Comment (eLBati) [inline __unported__/base_delivery_carrier_label/stock.py:313]: On 07/24/2014 01:53 PM, Sandy wrote:

> On the mailing list last year.
> https://lists.launchpad.net/openerp-community/msg03986.html
> 
> I put 120 so we wouldn't ignore line length entirely, and now it is at 
> least uniformly checked everywhere in OCA (no style splintering).
> 
> If we want to have the dicussion again and we come to a conclusion 
> that the line length is, say 100 (suggested in pep8 
> http://dinsdale.python.org/dev/peps/pep-0008/#maximum-line-length) 
> we have a constructive way of enforcing it.

If I remember correctly, on that occasion the majority said 79

Comment (bwrsandman) [inline __unported__/base_delivery_carrier_label/stock.py:313]: In that case, make a PR https://github.com/OCA/maintainer-quality-tools/edit/master/travis/travis_run_flake8 to set the line length to 80 (79 + '\n').

Comment (pedrobaeza) [inline __unported__/base_delivery_carrier_label/stock.py:313]: What @eLBati is saying it's true. I myself prefer 80 cols, but 100 is also correct. 120 is too long I think.

Regards.

Comment (bwrsandman) [inline __unported__/base_delivery_carrier_label/stock.py:313]: We can move this conversation to https://github.com/OCA/maintainer-quality-tools or the ML

Comment (pedrobaeza) [inline __unported__/base_delivery_carrier_label/stock.py:313]: Done:
https://github.com/OCA/maintainer-quality-tools/issues/46

**Pull request merged**